src/app/layouts/card/card-styling-sample/card-styling-sample.component.scss

```diff
@@ -0,0 +1,63 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+.card-wrapper {
+    max-width: 320px;
+    min-width: 220px;
+}
+
+$dark-card: igx-card-theme(
```

Comment:
Please include the following comment as in the other styling samples: /*BASIC THEMING*/

Reply:
Done